src/core/instance/state.js

```diff
@@ -147,7 +147,15 @@ function initComputed (vm: Component, computed: Object) {
 
   for (const key in computed) {
     const userDef = computed[key]
-    const getter = typeof userDef === 'function' ? userDef : userDef.get
+    let getter = typeof userDef === 'function' ? userDef : userDef.get
+    if (getter === undefined) {
+      warn(
+        `computed "${key}" is requires a "get" to be defined if a "set" is defined.` +
```

Comment:
What about saying: "No getter function has been defined for computed property `${key}`."